test/user_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+
+class UserTest < Minitest::Test
+
+  def setup
+    @sal = User.new("Sal")
+    @ali = User.new("Ali")
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+  end
+
+  def test_it_exists
+    assert_instance_of User, @sal
+  end
+
+  def test_user_has_name
+    assert_equal "Sal", @sal.name
+  end
+
+  def test_user_has_jokes
+    assert_equal [], @sal.jokes
+  end
```

Comment:
Good job up to here; feel free to group all initialized attributes into a single `test_it_has_attributes` test or something